### DISCUSSION TITLE
No water ripples far from world center

### DISCUSSION THREAD

**Discussion by EddieSoloProjects** [Sartaq12/WaterInteractionPlugin#8] [Ideas]
I have a Listen Server game that has a large world. It's 140 miles from world center to the beach. I'm seeing a small splash effect but no water ripples. The sample scene works fine in the same project. So either I am missing something or there is an issue far from world center.

**Comment by EddieSoloProjects** [comment]
So I just tested my character at 0,0,0 and it's what I thought. It works fine there. So there is an issue at large distances from world center. To reproduce place player 50-100 miles away from world center.

**Comment by Sartaq12** [comment]
Hello, I've tested it and the world space sims do break down at 100 miles away.
However this is true of any worldspace sim in FluidNinjaLIVE, therefore this is more a feature request for it than this plugin, as lack of support for large world coordinates is not due to anything contained in this plugin.

My course of action is to wait until they tackle it, then this plugin will automatically get the fix when upgrading versions.

**Reply by lSY-A** [reply]
您好 我想知道这个插件是否支持5.03？ 我想在5.0.3版本运行它

**Reply by Sartaq12** [reply]
No, it does not.

**Reply by lSY-A** [reply]
chinese : 噢 是的 我想问的的您是否可以提供一个5.1版本的插件 非常感谢！

**Reply by Sartaq12** [reply]
![image](https://github.com/Sartaq12/WaterInteractionPlugin/assets/37218094/0f3f831f-461d-4d4f-9bcf-43fd1269bfd7)
it's available in [Releases](https://github.com/Sartaq12/WaterInteractionPlugin/releases)
and there's a corresponding 5.1 branch on this repo

**Comment by EddieSoloProjects** [comment]
I just posted on Fluid Ninja Live's Market page requesting the addition.

**Reply by lSY-A** [reply]
您好 我想知道这个插件是否支持5.03？ 我想在5.0.3版本运行它

**Comment by Kynolin** [comment]
This is almost certainly a limitation of FluidNinja and not related to this plugin. I suggest closing this issue as unrelated. The issue should be resolved by the FluidNinja devs. (We don't have a public GitHub repo, so I can't link this to anything.)

**Comment by Sartaq12** [comment]
Thanks for confirming. I will convert this to a discussion for the time being, in case anyone comes along with a proposal or solution.

**Comment by EddieSoloProjects** [comment]
It's frustrating that a plugin for an Infinite ocean only works in a 5-10KM range. I asked the dev of Fluid Ninja and he said it could take 6 months for the fix.

**Reply by lSY-A** [reply]
是的 oceanlogy 5.03版本也出现了这个问题，但是得到了修复

**Comment by APinchofDill** [comment]
@EddieSoloProjects Not that it solves the direct issue but a plugin like Multiworld or rolling your own solution to async loading worlds could allow for instant loading when a player hits a certain distance to reset the distance and fix this limitation. Big oceans could be rendered in squares / secondary worlds.

**Comment by Sartaq12** [comment]
This issue has been solved :)
